internal/wrappers/generate_mock.sh

```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
It should be 2020-2022 as we are modifying it now.

Reply:
changed it

Reply:
Thank you.